data/languages/mips.sinc

```diff
@@ -550,8 +550,8 @@ define pcodeop syscall;
     }
 
     # 0010 11ss ssst tttt iiii iiii iiii iiii
-    :sltiu RT, RSsrc, simmed        		is prime=11 & RSsrc & RT & simmed {
-        RT = zext( RSsrc < simmed );
+    :sltiu RT, RSsrc, immed        		is prime=11 & RSsrc & RT & immed {
+        RT = zext( RSsrc < immed );
```

Comment:
So according to the docs, the unsigned 16 `immed` value is sign extended and then the comparison is done as unsigned integers.

Reply:
Ahh so thats what it meant. Changed back